Assets/Scripts/Player/PlayerController.cs

```diff
@@ -86,8 +89,11 @@ private void SetUpControls()
             _controls.Game.Move.performed += Move;
             _controls.Game.RotateCamera.performed += RotateCamera;
             _controls.Game.Zoom.performed += Zoom;
+            _controls.Game.Pause.performed += Pause;
         }
 
+        private void Pause(InputAction.CallbackContext obj) => GameManager.Instance.TogglePause();
+
         /// <summary>
         /// When the right mouse button is pressed, the player moves to the pressed location using a raycast and the NavMeshAgent.
```

Comment:
Getting this error when playing the game -> returning to main menu -> play again -> continue

```
The object of type 'Camera' has been destroyed but you are still trying to access it.
Your script should either check if it is null or you should not destroy the object. (line 102)
```

Issues detected: returning to main menu and clicking play again opens pause menu, player position is reset